scripts/templates/changelog-breaking-changes.ejs

```diff
@@ -0,0 +1,17 @@
+<%
+const breakingChanges = [];
+
+for (const commit of commits) {
+    const breakingChange = commit.notes.find(c => c.title === 'BREAKING CHANGE');
+    if (breakingChange) {
+        breakingChanges.push(breakingChange.text);
+    }
+}
+
+if (breakingChanges.length) {
+%>
+
+# Breaking Changes
+
+<%= breakingChanges.map(m => `- ${m}`).join('\n\n\n')  %>
```

Comment:
Commits with breaking changes will not necessarily have them as the first line of the commit. It is also hard to relate the description being shown currently to a specific commit.

WDYT of printing here the full commit information (e.g. commit with link, full commit title and body) instead of just the body?

Reply:
Yeah that's a good idea. Updated.